Documentation/ApiOverview/Database/DatabaseStructure/Index.rst

```diff
@@ -1,124 +1,141 @@
+
+..  index::
+    Tables; MM relations
+    Tables; Cache
+    Tables; System information
+..  _database-structure-other-tables:
 
 Other Tables
 ============
 
-The tables which are not managed via the TYPO3 CMS backend fill various
-roles. Some of the most common are:
-
-- MM relations: when tables are related using a many-to-many relationship,
-  another table must hold these relations. Examples are the table storing
-  relations between categories and categorized records
-  ("sys\_category\_record\_mm") or the table storing relations
-  between files and their various usages in pages, content elements, etc.
-  ("sys\_file\_reference"). The latter is an interesting example,
-  because it does actually appear in the backend, although only
-  as part of :ref:`inline records <t3tca:columns-inline>`.
-
-- Cache: when a cache is defined as using the database as a cache backend,
-  TYPO3 CMS will automatically create and manage the relevant cache tables.
-
-- System information: there exist tables storing information about sessions,
-  both frontend and backend ("fe\_sessions" and "be\_sessions" respectively),
-  a table for a central registry ("sys\_registry") and quite a few others.
-
-All these tables are not subject to the uid/pid constraint mentioned
-above, but they may have such fields if it is convenient for whatever
+The tables which are not managed through the TYPO3 backend serve various
+purposes. Some of the most common are:
+
+-   MM relations: When tables are connected via a many-to-many relationship,
+    another table must store these relations. Examples are the table storing
+    relations between :ref:`categories <categories>` and categorized records
+    (:sql:`"sys_category_record_mm`) or the table storing relations
+    between files and their various usages in pages, content elements, etc.
+    (:sql:`sys_file_reference`). The latter is an interesting example,
+    because it actually appears in the backend, although only as part of
+    :ref:`inline records <t3tca:columns-inline>`.
+
+-   Cache: If a :ref:`cache <caching>` is defined to use the database as a
+    :ref:`cache backend <caching-backend>`, TYPO3 automatically creates and
+    manages the relevant cache tables.
+
+-   System information: There are tables that store information about
+    :ref:`sessions <sessions>`, both frontend and backend (:sql:`fe_sessions`
+    and `be_sessions`  respectively), a table for a central registry
+    (:sql:`sys_registry`) and some others.
+
+All these tables are not subject to the :sql:`uid`/:sql:`pid` constraint
+mentioned above, but they may have such fields if it is convenient for some
 reason.
 
-There is no way such tables can be managed via the TYPO3 CMS
-backend unless a specific module provides a form of access to it.
-For example, the :guilabel:`System > Log` module provides an interface
-to browse records from the :sql:`sys_log` table.
+There is no way to manage such tables through the TYPO3 backend, unless a
```

Comment:
Well you manage the MM tables whenever you add or remove a relation?

Reply:
Now it's getting complicated ... "Well, you can use IRRE if you have MM tables or you can build a backend module when you have "regular" tables."

Reply:
But we can move the paragraph into the last list item, then it makes sense.

Reply:
Not sure if it makes sense to group MM tables with other internal tables as they have a special meaning and are managed in a different way then other internal tables